### DISCUSSION TITLE
Dark mode *auto*

### DISCUSSION THREAD

**Discussion by jgclark** [ttscoff/bunch-docs#110] [Ideas]
As well as being able to turn dark mode on and off, it would be cool to set it to `dark mode auto` or similar.  Sometimes my Mac gets into the 'wrong' mode for some reason, and without complex coding to sort out what time it is, it would be simpler to have an `auto` setting.
Thanks.

**Comment by ttscoff** [comment]
Bunch just uses AppleScript to toggle dark mode, and (unfortunately) the 
dark mode preference in System Events' scripting library is a boolean 
--- it's either true or false. The auto setting that Catalina added is 
not accessible via scripting, AFAICT.

You can use Bunch's conditional logic to set one or the other depending 
on time of day, but it's still setting it to "on" or "off," and not 
"auto."

-Brett

**Reply by ttscoff** [reply]
I did find the global preference for this, might be able to implement it. Will look at it when I get a chance.
